crates/networking/discv5/src/eth2.rs

```diff
@@ -0,0 +1,103 @@
+use alloy_primitives::{B256, aliases::B32, fixed_bytes};
+use ethereum_ssz::Encode;
+use ethereum_ssz_derive::{Decode, Encode};
+use ream_consensus::fork_data::ForkData;
+
+// Constants
+pub const FAR_FUTURE_EPOCH: u64 = 18446744073709551615;
+pub const GENESIS_FORK_VERSION: B32 = fixed_bytes!("0x00000000");
+pub const GENESIS_VALIDATORS_ROOT: B256 =
+    fixed_bytes!("0x0000000000000000000000000000000000000000000000000000000000000000");
+pub const ENR_ETH2_KEY: &str = "eth2";
+// Types
+#[derive(Default, Debug, Encode, Decode)]
```

Comment:
```suggestion
pub const ENR_ETH2_KEY: &str = "eth2";

#[derive(Default, Debug, Encode, Decode)]
```

make sure there is a line between these and we don't need the comment